app/src/main/res/values/colors.xml

```diff
@@ -20,5 +20,6 @@
     <color name="orange">#de5833</color>
     <color name="powderBlue">#50BFFF</color>
     <color name="midGreen">#3fa140</color>
+    <color name="transparent">#00000000</color>
```

Comment:
Is this used?

Reply:
This was used while i was experimenting with getting the toolbar to be transparent. Not any more; good spot.